exec/src/main/java/com/optum/sourcehawk/exec/FlattenCommand.java

```diff
@@ -0,0 +1,74 @@
+package com.optum.sourcehawk.exec;
+
+import com.optum.sourcehawk.core.scan.FlattenResult;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import picocli.CommandLine;
+
+/**
+ * CLI entry point for executing Sourcehawk flatten command
+ *
+ * @author Christian Oestreich
+ */
+@Slf4j
+@CommandLine.Command(
+        name = FlattenCommand.COMMAND_NAME,
+        aliases = {"flat", "merge", "converge"},
+        description = "Flatten the sourcehawk.yml onto system by recursively reading and merging local and remote configurations"
+)
+class FlattenCommand extends AbstractCommand {
+
+    static final String COMMAND_NAME = "flatten";
+
+
+    @CommandLine.Option(
+            names = {"-o", "--output"},
+            description = "Override the default output name and path flattened file",
+            defaultValue = "sourcehawk-flattened.yml",
+            showDefaultValue = CommandLine.Help.Visibility.ALWAYS
+    )
+    private String output;
+
+    @CommandLine.Option(
+            names = {"-d", "--dry-run"},
+            description = "Display output to console instead of file system"
+    )
+    private boolean dryRun;
```

Comment:
Maybe this isn't needed if we default to `stdout` above

Reply:
removed